runner-charts/deployment/templates/deployment.yaml

```diff
@@ -47,3 +48,38 @@ spec:
             - name: {{ .name }}
               value: {{ .value }}
         {{- end }}
+# Enable Cluster Wide Proxy
+# https://docs.openshift.com/container-platform/4.6/networking/enable-cluster-wide-proxy.html
+            {{ if .Values.clusterProxy }}
+            - name: https_proxy
+              value: {{ required ".Values.https_proxy must be set if .Values.clusterProxy enabled." .Values.https_proxy }}
+            - name: http_proxy
+              value: {{ required ".Values.http_proxy must be set if .Values.clusterProxy enabled." .Values.http_proxy }}
+            - name: no_proxy
+              value: {{ required ".Values.no_proxy must be set if .Values.clusterProxy enabled." .Values.no_proxy }}
+            {{ end }}
+
+# Enable custom cluster PKI
+# https://docs.openshift.com/container-platform/4.6/networking/configuring-a-custom-pki.html
+{{ if .Values.clusterPKI }}
+          volumeMounts: 
+            - name: trusted-ca
+              mountPath: /etc/pki/ca-trust/extracted/pem
+              readOnly: true 
+
+      volumes: 
+        - name: trusted-ca
```

Comment:
Can you put `trusted-ca` into a `value`? Or maybe even two values? One for the config map name, and one for the volume name?

Reply:
The cluster wide custom PKI is imported via an operator, all you have to do is to give a label to it and the operator updates.
https://docs.openshift.com/container-platform/4.6/networking/configuring-a-custom-pki.html

I agree that the configmap should be a separate file, but I wouldn't like to pass the trusted-ca through since Openshift has this managed for us.

Once you restructure the chart I can add these for review.